services/web/client/source/class/qxapp/dev/fake/Fake.js

```diff
@@ -6,7 +6,7 @@
 
 /* global window */
 
-qx.Class.define("qxapp.data.Fake", {
+qx.Class.define("qxapp.dev.fake.Data", {
```

Comment:
Is this file duplicated?

Reply:
No. this is one of the outlines in the description. ``qxapp.data.Fake`` contains mock data so it was **renamed and moved** into ``qxapp.dev`` namespace. Be careful from now on

Reply:
My bad then. The way Github shows the "Files changed" confused me.